File: beginner/1159.py
outputs = []

while True:
    x = int(input())
    if x == 0:
        break;
    i = 0
    resultado = 0
    while i < 5:
        if x % 2 == 0:
            i+=1
            resultado += x
        x+=1
    outputs.append(resultado)

for output in outputs:
    print(output)

File: beginner/1159.c
# include<stdio.h>

int main(){

	int x, resultado, lista;
	x = resultado = lista = 0;

	int outputs[100] = {};

	while(1){
		scanf("%i", &x);

		if(x == 0)
			break;
		resultado = 0;

		for(int i = 0 ; i < 5 ; ){
			if( ( x % 2) == 0 ){
				resultado = resultado + x;
				++i;
			}
			++x;
		}
		outputs[lista++] = resultado;

	}

	for( int i = 0 ; i < lista ; ++i )
		printf("%i\n", outputs[i]);


	return 0;
}
